.env

```diff
@@ -1,4 +1,9 @@
-# On running the 'build' commands this file is replaced by the files in `env-config/`
-# To ignore and keep this file we have a command in `postshrinkwrap` which assumes this file is not changing	
-# The command is `git update-index --assume-unchanged .env`
-# Source: https://wildlyinaccurate.com/git-ignore-changes-in-already-tracked-files/
\ No newline at end of file
+## NB only variables prefixed with SIMORGH_ are available on both the server and client
```

Comment:
Hey Lily - we don't want to update this file. Can you revert it and re-push?

Once you've done that, some combination of 

```
git update-index --no-assume-unchanged .env
git reset .env    
git update-index --assume-unchanged .env  
```

Should prevent it happening in future.

Reply:
Do you know why this has happened when I didn't touch the file?

Reply:
Do I just replace it with:
# On running the 'build' commands this file is replaced by the files in `env-config/`
# To ignore and keep this file we have a command in `postshrinkwrap` which assumes this file is not changing	
# The command is `git update-index --assume-unchanged .env`
# Source: https://wildlyinaccurate.com/git-ignore-changes-in-already-tracked-files/

Reply:
wow font

Reply:
I think there's an NPM command that changes the file which you must have ran - the important bit is making git ignore it so that when his command is ran and the .env file is changed, those changes aren't staged.

Reply:
npm run test?